### PR TITLE
Add the new header for GoCD service API

### PR DESCRIPTION
**Pull request by zmingxie**
In the latest GoCD release, a new HTTP header is added in its material notification API: https://api.go.cd/current/#notify-git-materials. As a result, the current GoCD github-services is broken. Adding `http.headers['confirm'] = true` will fix this. (I have tested it in my GitHub Enterprise instance)

(This patch also converts line-endings to Unix-style)

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
@zmingxie Can you separate the line ending changes into another commit? It's really hard to follow this otherwise. Thanks!

**Comment by MikeMcQuaid** [comment]
That said, this is something we consider more than a bug fix as it's a change GoCD has made rather than a bug in our code: https://github.com/github/github-services/blob/master/.github/CONTRIBUTING.md#updating-an-existing-service so I'm not sure if we'll merge this.

**Comment by zmingxie** [comment]
@MikeMcQuaid I just back out the line-endings conversion, so you can see this is just a simple one line change. (My editor converted those line-endings automatically earlier, sorry about that!)

I understand these github services are no longer actively maintained, but I've confirmed this is backward compatible with earlier versions of GoCD. On the other hand, thanks to GitHub Supports help, I've patched this in my GitHub Enterprise server. Just want to share this patch in case other people face the same issue after upgrading their GoCD server..

**Comment by MikeMcQuaid** [comment]
Thanks @zmingxie!
